### PR TITLE
fixed highlighting of active tab in menu

### PR DESCRIPTION
**Pull request by plata**
Before, the tab lost its highlighting once it lost focus (e.g. if you clicked any button).

highlighting

### CODE REVIEW

File: phoenicis-javafx/src/main/resources/com/playonlinux/javafx/views/common/defaultTheme.css [hunk 1]
```diff
@@ -122,6 +122,14 @@
     -fx-max-height: 2.5em;
 }
 
+#menuPane > .tab-header-area .tab:selected {
```
Comment (qparis) [inline phoenicis-javafx/src/main/resources/com/playonlinux/javafx/views/common/defaultTheme.css]: Can you try 
#menuPane > .tab-header-area .tab:selected {
    -fx-background-color: linear-gradient(#529bda, #2173A5);
    -fx-focus-color: transparent;
    -fx-faint-focus-color: transparent;
}
instead?

Comment (plata) [inline phoenicis-javafx/src/main/resources/com/playonlinux/javafx/views/common/defaultTheme.css]: Looks like this:
gradient

Comment (qparis) [inline phoenicis-javafx/src/main/resources/com/playonlinux/javafx/views/common/defaultTheme.css]: Do you like it?

Comment (plata) [inline phoenicis-javafx/src/main/resources/com/playonlinux/javafx/views/common/defaultTheme.css]: Fine for me. What colors would you use for the dark one?

Or how about something like KDE:
KDE like

Btw: Do you see the ugly frame around the left panel with the dark theme?

Comment (qparis) [inline phoenicis-javafx/src/main/resources/com/playonlinux/javafx/views/common/defaultTheme.css]: I like your idea. The frame problem has been fixed in one of my last PR

**Pull request merged**